README.md

```diff
@@ -79,6 +89,21 @@ KeyFactory algorithms:
 * RSA
 * ACCP's SecureRandom uses AWS-LC's DRBG implementation.
 
+# Notes on ACCP-FIPS
+ACCP-FIPS is a variation of ACCP which is uses AWS-LC built in FIPS-mode.
+AWS-LC is undergoing FIPS validation testing by an accredited lab and,
+upon completion, will be submitted to NIST for certification.
+This process can take a long time, so we provide ACCP-FIPS for
+experimentation purposes in the interim.
+
+Version 2.3.0 is the first release of ACCP-FIPS. The Maven coordinates for
+ACCP-FIPS is the same as ACCP with one difference that ACCP-FIPS's
```

Comment:
"is the same" --> "are the same" (plural agreement with "coordinates")

Reply:
done